### ISSUE TITLE
Issue Launching Gazebo Simulation

### ISSUE THREAD

**Issue by Ahmad8864** [purdue-arc/arc_robot_arm#21]
**System Info (please complete the following information):**
 - OS/version: MacOS Big Sur 11.5.2

**Error Log**
Resource not found: gazebo_ros
ROS path [0]=/usr/local/Caskroom/miniforge/base/envs/ros_env/share/ros
ROS path [1]=/Users/Ahmad/catkin_ws/src/arc_robot_arm/chess_piece_detector
ROS path [2]=/Users/Ahmad/catkin_ws/src/arc_robot_arm/chessboard_gazebo
ROS path [3]=/Users/Ahmad/catkin_ws/src/arc_robot_arm/protoarm_bringup
ROS path [4]=/Users/Ahmad/catkin_ws/src/arc_robot_arm/protoarm_description
ROS path [5]=/Users/Ahmad/catkin_ws/src/arc_robot_arm/protoarm_driver
ROS path [6]=/Users/Ahmad/catkin_ws/src/arc_robot_arm/protoarm_gazebo
ROS path [7]=/Users/Ahmad/catkin_ws/src/arc_robot_arm/protoarm_kinematics
ROS path [8]=/Users/Ahmad/catkin_ws/src/arc_robot_arm/protoarm_moveit_config
ROS path [9]=/Users/Ahmad/catkin_ws/src/arc_robot_arm/protoarm_visual_servoing
ROS path [10]=/Users/Ahmad/catkin_ws/src/arc_robot_arm/realsense_ros_gazebo
ROS path [11]=/Users/Ahmad/catkin_ws/src/robot_car_description
ROS path [12]=/Users/Ahmad/catkin_ws/src/arc_robot_arm/yolov5_pytorch_ros
ROS path [13]=/usr/local/Caskroom/miniforge/base/envs/ros_env/share
The traceback for the exception was written to the log file

**To Reproduce**
1. run "roslaunch protoarm_bringup sim.launch"
2. See error

**Context**
I'm trying to launch the robot in gazebo simulation

**Screenshots**

**Comment by raghavauppuluri13** [comment]
@Ahmad8864 Fixed by #22. Try running:
```
cd path/to/root_workspace_dir
rosdep install --from-paths src --ignore-src --rosdistro=noetic -y
```

**Comment by raghavauppuluri13** [comment]
Found the issue and fixed with #23. Made a silly error by not changing the dashes to underscores in the package.xml
